migrations/2024-08-29-132507_add-leaderboards-table/up.sql

```diff
@@ -0,0 +1,29 @@
+-- Your SQL goes here
+CREATE TABLE leaderboard (
+    id SERIAL PRIMARY KEY,
+    user_id UUID NOT NULL REFERENCES users(id),
+    achievements JSONB,
+    score INT NOT NULL DEFAULT 0,
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP NOT NULL DEFAULT now()
```

Comment:
This will be set to datetime at time of row creation and will need to be updated manually on every update after that (unless we add a database trigger).  Just something to note.